### PR TITLE
Remove whitespace and convert the domain to lowercase before check

### PR DESCRIPTION
**Pull request by Sothatsit**
Hello,

Thank you for making this dataset into an NPM library!

I thought that the check for disposable domains might be made more robust by adding the following preprocessing steps:

1. Remove leading/trailing whitespace.
2. Convert the domain to lowercase.

This would help anyone that is passing raw user input into the function.

Thanks,
~ Paddy

### CODE REVIEW

**Review by mziyut — APPROVED** [review]
LGTM!

File: test/index.test.ts [hunk 1]
```diff
@@ -4,12 +4,14 @@ describe('disposableEmailBlocklist', () => {
   test('should return an array of strings', () => {
     const result = disposableEmailBlocklist();
 
-    expect(result).not.toEqual([]);
+    expect(Array.isArray(result)).toBe(true);
+    expect(result.length).toBeGreaterThan(0);
+    expect(result.every(value => typeof value === 'string')).toBe(true);
```
Comment (mziyut) [inline test/index.test.ts:9]: Prettier has failed, can you just fix this part?

```suggestion
    expect(result.every((value) => typeof value === 'string')).toBe(true);
```

This command will automatically correct the problem.

```sh
> npm run lint:fix
```

**Review by mziyut — CHANGES_REQUESTED** [review]
I'm sorry, but I just need you to fix one thing.

File: test/index.test.ts (hunk 1 above)
Comment (Sothatsit) [reply test/index.test.ts]: I've added a new commit for this! 👍

Comment (mziyut) [reply test/index.test.ts]: Thank you!

**Review by mziyut — APPROVED** [review]

**Pull request merged**